src/modules/authentication.js

```diff
@@ -0,0 +1,19 @@
+import auth from "./auth";
```

Comment:
you can call the file name in pascal case, same name as the object is exporting: "Authentication"

Reply:
Ok